te/te_docker/Dockerfile

```diff
@@ -118,7 +120,7 @@ RUN chmod -R 755 /var/www/html/
 RUN echo "Done TE Dockerization!\n"
 
 ENTRYPOINT sed -i "s/5001/$NGINX_PORT/g" /etc/nginx/sites-available/default && \
-    redis-server --port $REDIS_PORT --daemonize yes && /etc/init.d/nginx restart && \
+    redis-server --port $REDIS_PORT --protected-mode no --daemonize yes && /etc/init.d/nginx restart && \
```

Comment:
the default value of protected mode has changed in the latest ubuntu version, causing failure to connect to te_dp machines.